octopoes/bits/spf_discovery/spf_discovery.py

```diff
@@ -18,6 +19,10 @@
 def run(input_ooi: DNSTXTRecord, additional_oois: list, config: dict[str, Any]) -> Iterator[OOI]:
     if input_ooi.value.startswith("v=spf1"):
         spf_value = input_ooi.value.replace("%(d)", input_ooi.hostname.tokenized.name)
```

Comment:
As far as I can see this is also not correct, see https://www.rfc-editor.org/rfc/rfc7208#section-7.3

```suggestion
        spf_value = input_ooi.value.replace("%{d}", input_ooi.hostname.tokenized.name)
```